fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfLexicon.java

```diff
@@ -263,12 +233,12 @@
         managedProperties = b.build();
     }
 
-    public static final Set<Property> relaxableProperties
+    private static final Set<Property> relaxableProperties
```

Comment:
This is the exact same thing as `serverManagedProperties` except for the order of the properties. Perhaps we should delete `serverManagedProperties` and use this in its place?

Reply:
Although the values are the same, there is a semantic difference. I would be inclined to create a separate ticket the reconciles our various incarnations of "Server Managed" variables.

Reply:
I believe these "clean-up" PRs should try to eliminate any possibility of introducing bugs.

New ticket: https://jira.duraspace.org/browse/FCREPO-2919